book/src/building/swiftc-and-cargo/README.md

```diff
@@ -163,9 +163,7 @@ fn main() {
 #!/bin/bash
 set -e
 
-# The swift-bridge CLI does not exist yet. Open an issue if you need to use
-# this approach and I'll happily whip up the CLI.
-swift-bridge -f src/lib.rs > generated
+swift-bridge-cli parse-bridges <package name> src/lib.rs
```

Comment:
Let's use `--file / -f` for each source file:

Let's use `--crate-name` for the crate name:

Reasons:

- allow arguments to be specified in any order
- more clear which argument is which. Right now `<package name>` and `<file name>` are right next to each other without any flags, making it a little harder to understand what each argument does.

Reply:
Done; updates the docs as well